com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/TemporalAntiAliasing.shader

```diff
@@ -69,6 +69,19 @@ Shader "Hidden/HDRP/TemporalAA"
     #define PERCEPTUAL_SPACE 1
     #define PERCEPTUAL_SPACE_ONLY_END 0 && (PERCEPTUAL_SPACE == 0)
 
+#elif defined(POST_DOF)
+    #define YCOCG 1     
+    #define HISTORY_SAMPLING_METHOD BILINEAR
+    #define WIDE_NEIGHBOURHOOD 0
+    #define NEIGHBOUROOD_CORNER_METHOD VARIANCE
+    #define CENTRAL_FILTERING NO_FILTERINGs
+    #define HISTORY_CLIP DIRECT_CLIP
+    #define ANTI_FLICKER 1
+    #define ANTI_FLICKER_MV_DEPENDENT 1
+    #define VELOCITY_REJECTION (defined(ENABLE_MV_REJECTION) && 0)
+    #define PERCEPTUAL_SPACE 1
+    #define PERCEPTUAL_SPACE_ONLY_END 0 && (PERCEPTUAL_SPACE == 0)
```

Comment:
Didn't we want to disable the sharpening too?

Reply:
I had a version with the post-dof sharpening disabled, but in the end it is not what we want (it was "reverting" the pre-dof sharpening). 

The right thing to do is to perform sharpening only the last time TAA will run, but I might have to introduce a new variant to implement that (because it means that medium TAA might need to run without sharpening, depending on the DoF settings!). As it is, I don't get any artifacts (as long as we don't do two times the 5tap bicubic), so I left this as a low priority, until I resolve another bug.

Reply:
:+1: